blurp-app/src/pages/testpage.jsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect } from "react";
+import MultiUndirectedGraph from "graphology";
+import { SigmaContainer, useLoadGraph } from "@react-sigma/core";
+import "@react-sigma/core/lib/react-sigma.min.css";
+import { v4 as uuidv4 } from 'uuid';
+
+export const LoadGraph = () => {
+  const loadGraph = useLoadGraph();
+  const node1 = uuidv4();
+  const node2 = uuidv4();
+
+  useEffect(() => {
+    //i'm leaving these settings "exposed" for future tweeks if necesssary
+    //we can also use pre-defined graph types: MultiGraph, MultiUndirectedGraph, etc.
+    // const graph = new Graph({type: "mixed", allowSelfLoops: false, multi: true});
+    const graph = new MultiUndirectedGraph();
+
+    //test add Node & Edges
+    graph.addNode("A", { x: 0, y: 0, label: "A", size: 15, color: "#FA4F40" });
```

Comment:
Should the Nodes id also be uuidv4() as well? Obviously, the label should be 'A' or the actual name of the person but don't we want to have the id of the node unique as well.

Reply:
That would be a good idea. We'll need to convert uuidv4 output into a string via a promise function instead of the "A" or "B" parameter. I'll put in a ticket for it.